lib/src/tag_detection.dart

```diff
@@ -502,6 +512,12 @@ class _DeclaredFlutterPlatformDetector {
 
   _DeclaredFlutterPlatformDetector(this._pubspecCache);
 
+  bool _isFlutterPlugin(Uri uri) =>
+      uri.scheme != 'package' ||
+      _declaredFlutterPlatforms(_pubspecCache.packageName(uri)) != null;
```

Comment:
I suppose there is a need to distinguish between "_package_" and "_plugin without any supported platforms_", right?

Reply:
These are the "declared" platforms. A non-plugin has no declared platforms.

Reply:
And a plugin with no declared platforms would return the empty set.